subjects/java-piscine/Quest1/01-02-int-operation/README.md

```diff
@@ -0,0 +1,76 @@
+## IntOperations
```

Comment:
if you use `int-operations` in the folder naming you should use `int-operations` as title instead of `IntOperations`
pick one and be consistent for every exercise, in Java, if I'm not mistaken the naming convention that is mostly used is the camelCase so i would go with that...